src/app.jsx

```diff
@@ -368,6 +368,7 @@ const NavigatorCardBody = ({
     sortBy,
 }) => {
     const [boxPerRow, setBoxPerRow] = useState(0);
+    const Dialogs = useDialogs();
```

Comment:
"disable keyboard input if a dialog is active": please add a justification to the commit message. It says "what", but not "why" or "when does this happen" or "what does this break/fix". Would also be good to illustrate/prove that with a test.

Reply:
Updated!
As for the test, I will add a case for it once I figure out how to fix using `b.key_press` with the arrow keys :/

Reply:
Please fix the overly long line in the commit message. This can of course also happen with squash&merge. The justification is good now, thanks!